spec/integration/guest_view_spec.rb

```diff
@@ -107,13 +107,27 @@
 
       expect(current_path).to eq(root_path)
     end
+
+    it "does not allow guest access to the admin page" do
+      visit admin_path
+
+      expect(current_path).to eq(root_path)
+    end
+
+    it "does not allow guest access to user pages" do
+      user = create(:user)
+      visit admin_show_user_path(user.id)
+
+      expect(current_path).to eq(root_path)
+    end
   end
 
   def create_item
     image = create(:image)
-    item = create(:item, title: "milk",
-                                     description: "some cheese stuff",
-                                     image_id: image.id)
+    item = create(:item,
```

Comment:
Have you pulled from the most recent master yet? I think may not pass now do to the update validations set on Item.